File: ruby/raindrops/raindrops.rb

module BookKeeping   
    VERSION = 3
end
  
  
class Raindrops
    
    def self.convert(num)
      str = ""
      str += 'Pling' if num % 3 == 0 
      str += 'Plang' if num % 5 == 0 
      str += 'Plong' if num % 7 == 0 
      str = num.to_s if str == ""  
      return str  
    end
    
end

File: javascript/raindrops/raindrops.js

class Raindrops {
  
  convert(num){
    let str = ''
    if (num % 3 === 0) {str += 'Pling'}
    if (num % 5 === 0) {str += 'Plang'}
    if (num % 7 === 0) {str += 'Plong'}
    if (str === '') { str = num.toString() }
    return str
  }
}

module.exports = Raindrops;